cypress/integration/InstancesTest.ts

```diff
@@ -272,6 +272,82 @@ describe("Instances Test", () => {
   });
 
   onlyOn(isEnvironmentType(EnvType.Mocked), () => {
+    describe("Instance Error Handler", () => {
+      it("View error handler details", () => {
+        // visit 'Instance one'
+        cy.visit("/instance/3543edaa-1851-4ad7-96be-ebde7d20d717");
+
+        cy.ouiaId("error-handling", "PF4/TabButton").click();
+
+        cy.ouiaId("error-handling-section", "PF4/Text")
+          .should("contain.text", "Error handling method")
+          .should("be.visible");
+
+        cy.ouiaType("ProcessorConfig/FormGroup").should("have.length", 5);
+
+        cy.ouiaId("error_handling_method", "ProcessorConfig/FormGroup").within(
+          () => {
+            cy.get("[data-testid='error_handling_method']").should(
+              "contain.text",
+              "Error handling method"
+            );
+            cy.get("[data-testid='error_handling_method-value']").should(
+              "contain.text",
+              "Webhook"
+            );
+          }
+        );
+
+        cy.ouiaId("endpoint", "ProcessorConfig/FormGroup").within(() => {
+          cy.get("[data-testid='endpoint']").should("contain.text", "Endpoint");
+          cy.get("[data-testid='endpoint-value']").should(
+            "contain.text",
+            "http://google.com"
+          );
+        });
+
+        cy.ouiaId("basic_auth_username", "ProcessorConfig/FormGroup").within(
+          () => {
+            cy.get("[data-testid='basic_auth_username']").should(
+              "contain.text",
+              "Basic Auth Username"
+            );
+            cy.get("[data-testid='basic_auth_username-value']").should(
+              "contain.text",
+              "user"
+            );
+          }
+        );
+
+        cy.ouiaId("basic_auth_password", "ProcessorConfig/FormGroup").within(
+          () => {
+            cy.get("[data-testid='basic_auth_password']").should(
+              "contain.text",
+              "Basic Auth Password"
+            );
+            cy.get("[data-testid='basic_auth_password-value']").should(
+              "contain.text",
+              "****"
+            );
```

Comment:
IMO, Password must be asserted with 'have' as this value is constant

Reply:
Same opinion for other fields as well.

Reply:
pushing in a separate commit, thank you